test/coverband/collectors/coverage_test.rb

```diff
@@ -70,4 +70,38 @@ def teardown
     heroku_build_file = '/tmp/build_81feca8c72366e4edf020dc6f1937485/config/initializers/assets.rb'
     assert_equal false, @coverband.send(:track_file?, heroku_build_file)
   end
+
+  test 'one shot line coverage disabled for ruby >= 2.6' do
+    return unless Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.5.0')
+    Coverband::Collectors::Coverage.expects(:ruby_version_greater_than_or_equal_to?).with('2.6.0').returns(true)
+    ::Coverage.expects(:running?).returns(false)
+    ::Coverage.expects(:start).with(oneshot_lines: false)
+    Coverband::Collectors::Coverage.send(:new)
+  end
+
+  test 'one shot line coverage enabled for ruby >= 2.6' do
+    return unless Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.5.0')
+    Coverband.configuration.expects(:use_oneshot_lines_coverage).returns(true)
+    Coverband::Collectors::Coverage.expects(:ruby_version_greater_than_or_equal_to?).with('2.6.0').returns(true)
+    ::Coverage.expects(:running?).returns(false)
+    ::Coverage.expects(:start).with(oneshot_lines: true)
+    Coverband::Collectors::Coverage.send(:new)
+  end
+
+  test 'one shot line coverage for ruby >= 2.6 when already running' do
+    return unless Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.5.0')
+    Coverband::Collectors::Coverage.expects(:ruby_version_greater_than_or_equal_to?).with('2.6.0').returns(true)
+    ::Coverage.expects(:running?).returns(true)
+    ::Coverage.expects(:start).never
+    Coverband::Collectors::Coverage.send(:new)
+  end
+
+  test 'no one shot line coverage for ruby < 2.6' do
+    return unless Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.5.0')
+    Coverband::Collectors::Coverage.expects(:ruby_version_greater_than_or_equal_to?).with('2.6.0').returns(false)
+    Coverband::Collectors::Coverage.expects(:ruby_version_greater_than_or_equal_to?).with('2.5.0').returns(true)
+    ::Coverage.expects(:running?).returns(false)
+    ::Coverage.expects(:start).with()
+    Coverband::Collectors::Coverage.send(:new)
```

Comment:
Instead of invoking a private method, could we just calling `Coverage.reset_instance`?

Reply:
`reset_instance` doesn't seem to actually call `initialize` for a new instance so in this case, I don't think exercises the code this test wants to exercise.